views/sign-out.html

```diff
@@ -50,64 +51,63 @@
           }
         },
         hint: {
-            text: hintText
+         text: hintText
         },
         items: [
           {
             value: "yes",
             text: "Yes",
             attributes: {
-              "data-event-id": "yes-sign-out"
-            }
+            "data-event-id": "yes-sign-out"
```

Comment:
we've lost some of the indentation in this file, eg line 61, 68, 75-77 and a few more..
could you re-indent them and correct indention in the rest of this file? (views/sign-out.html)

Reply:
Indented this appropriately.